docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+
+> Download - If you haven't downloaded Visual Studio Code yet, click [here](/download) to give it a try. 
+
+## Install Python Extension
+
+Visual Studio Code is a fast editor and ships with only the basic features. Customize Visual Studio Code with Python by installing one of the popular Python Extensions. 
+
+1. Select an extension. 
+2. Install the extension by typing `ext install` into the command palette `kb(workbench.action.showCommands)`. 
+
+<div class="marketplace-extensions-python"></div>
+
+> Tip: these extensions are dynamically queried. Read the description and reviews to decide which extension is best for you. See more in the [Marketplace](https://marketplace.visualstudio.com/vscode).
+
+The examples in this document will use [Don Jayamanne's Python Extension](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python) as it has the highest number of downloads and the most features. 
+
+## Code Completion
+
+[Intellisense](/docs/editor/editingevolved#_intellisense) is intelligent code completion (in-context method and variable suggestions) across all your files and for built-in and thirty-party modules. 
+
+Quickly see methods, class members, and documentation.
+
+<video id="python-code-completion-video" src="/images/python_python-intellisense.mp4" poster="/images/python_python-intellisense-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: trigger code completion with `kb(editor.action.triggerSuggest)`.
+
+## Linting
+
+Linting is the analysis of your Python code for potential errors. Use Visual Studio Code to quickly navigate to the errors and warnings in your code. 
+
+<video id="python-linting-video" src="/images/python_python-linting.mp4" poster="/images/python_python-linting-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: [Don Jayamanne's Python extension](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python) gives you the option of using three different linters - [Pylint](https://www.pylint.org/), [Pep8](https://pypi.python.org/pypi/pep8), and [Flake8](https://flake8.readthedocs.io/en/latest/). See the [wiki](https://github.com/DonJayamanne/pythonVSCode/wiki/Linting) for more details. 
+
+## Debugging
+
+No more `print` statement debugging! Set breakpoints, watch variables, inspect data, and use the debug console. Debug a number of different type of Python applications, including multi threaded, web, and remote applications. 
+
+<video id="python-debugging-video" src="/images/python_python-debugging.mp4" poster="/images/python_python-debugging-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: Follow the instructions in the [wiki](https://github.com/DonJayamanne/pythonVSCode/wiki/Debugging) for getting started with debugging, including setting up your `launch.json` and troubleshooting common issues. 
+
+> Tip: Read more about general information about debugging in Visual Studio Code in the [debugging document](/docs/editor/debugging). 
```

Comment:
If you give the full Markdown filename /docs/editor/debugging.md, folks can navigate in GitHub.